common/static/common/js/spec/discussion/view/discussion_thread_view_spec.js

```diff
@@ -182,15 +183,17 @@
                     el: $('#fixture-element'),
                     course_settings: DiscussionSpecHelper.createTestCourseSettings()
                 });
+                spyOn($.fn, 'focus');
             });
             describe('responses', function() {
                 it('can post a first response', function() {
                     renderWithTestResponses(this.view, 0);
                     postResponse(this.view, 1);
                     expect(this.view.$('.forum-response').length).toBe(1);
                     expect(this.view.$('.post-actions-list').find('.action-edit').parent('.is-hidden').length).toBe(1);
-                    return expect(this.view.$('.response-actions-list').find('.action-edit')
+                    expect(this.view.$('.response-actions-list').find('.action-edit')
                         .parent().not('.is-hidden').length).toBe(1);
+                    return expect(this.view.$('.forum-response').focus).toHaveBeenCalled();
```

Comment:
Spying the jquery focus method and asserting it was run doesn't necessarily tell us if the element has been focused. I think a better test would be checking `document.activeElement === this.view.$('.forum-response')[0]`